src/scs/basics.md

```diff
@@ -0,0 +1,33 @@
+# Basic Security Measures
+
+## Source code / Version control security
+
+Briefly, the source code security is based on the fact that the source code is Two-person reviewed, version controlled, the history is verified, retained indefinitely. 
+
+### Commit signing
+All the commits to repositories must be GPG-signed. This can be achieved by enabling GPG commit signatures in the config:
```

Comment:
@vilvo & @jenninikko Should we open this abbreviations as well? I mean GPG, TLS, TCP?

Reply:
we should at least once, however, i think we should maintain a glossary in the appendix for all acronyms. then the question is if we should create links to that entry in the glossary when an abbreviation is encountered for the first time on a page/?

Reply:
The glossary was in my plans. We can stick to the rule as when you use any acronym or a term for the first time, you make a link to the glossary (glossary.md). Then in the glossary itself, in addition to the definition, it will be possible to provide links to third-party sources for additional information, not to overload sections.

I also suggest not making terms out of obvious abbreviations, such as OS -- operating systems.

This information will be added to some guidelines.